rasa/cli/scaffold.py

```diff
@@ -175,15 +185,21 @@ def run(args: argparse.Namespace) -> None:
             "Now let's start! 👇🏽\n".format(DOCS_BASE_URL)
         )
 
-    path = (
-        questionary.text(
-            "Please enter a path where the project will be "
-            "created [default: current directory]",
-            default=".",
+    if args.init_dir is not None:
+        path = args.init_dir
+    else:
+        path = (
+            questionary.text(
+                "Please enter a path where the project will be "
+                "created [default: current directory]",
+                default=".",
+            )
+            .skip_if(args.no_prompt, default=".")
+            .ask()
         )
-        .skip_if(args.no_prompt, default=".")
-        .ask()
-    )
+
+    if args.no_prompt and not os.path.isdir(path):
+        print_init_path_not_found()
```

Comment:
We might as well call the `print_error_and_exit()` right here as nothing more happens in `print_init_path_not_found()`

Reply:
it would also be useful to print that path in the error message, e.g. with 
```python
print_error_and_exit(f"Project init path '{path}' not found.")
```